qtp_sequencing/summary.py

```diff
@@ -136,6 +136,8 @@ def _summary_not_demultiplexed(artifact_type, filepaths):
     errors = []
     df = None
     for fps_type, fps in sorted(filepaths.items()):
+        if fps_type in {'html_summary'}:
```

Comment:
if this will always be just a one item set, wouldn't it be better to use == on 'html_summary' instead?

Reply:
Accepting this, since it is unambiguous what we're trying to do.